x-pack/legacy/plugins/ml/public/data_frame_analytics/common/analytics.ts

```diff
@@ -81,6 +82,15 @@ export const getDependentVar = (analysis: AnalysisConfig) => {
   return depVar;
 };
 
+export const getPredictionFieldName = (analysis: AnalysisConfig) => {
+  // Default to dependent_variable
```

Comment:
The comment here says `Default to dependent_variable` but I don't see that happening, is this comment meant as a TODO or does it happen somewhere else?

Reply:
I'd also suggest having all comments as [tsdoc](https://github.com/microsoft/tsdoc) for functions. More convenient to look up in IDE and let us generate the code documentation

Reply:
Good one @walterra - I've clarified this comment in 695aaeda022ae742baf164bbff52271069ee2b0b

I just wanted to make it clear that if prediction_field_name is not defined/set it will be defaulted to dependent_variable when the config is created. 

@darnautov - good suggestion, though in this case it's just more of an FYI comment for where this is used. Also I think TypeScript kind of accomplishes the same functionality by defining the parameters, return value, etc. 🤔